### ISSUE TITLE
Bot OFF

### ISSUE THREAD

**Issue by SHlupin** [udit-001/ado-bot#10]
Bot is in OFF mode; NOT WORKING as of 22 Feb 12: 27 PM IST

**Comment by udit-001** [comment]
Yeah, I am aware of that, I have currently decided not to run the bot anymore since I don't have the resources to keep it running. It won't be back up until I figure out how to run this bot in a much more economical manner.

**Comment by talhajavaidmalik** [comment]
[![Banner](https://pbs.twimg.com/media/GFz-XdoaUAAeB3h.jpg:large)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Greetings, crypto enthusiasts! Blastoff.zone has an irresistible offer for the first 10 participants! Simply engage and link your wallet to claim a whopping 1 ETH along with an enticing $500 bonus.

## How to Participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to head to the Blastoff.zone platform.
2. Link your Ethereum wallet on the platform to qualify for the $500 bonus.
3. Utilize the "Claim Now" feature to secure your tokens.

Don't let this golden opportunity slip away! Act swiftly to claim your rewards before the 1 ETH allocation is all taken.

Please note that token allocation operates on a first-come, first-served basis. Out of the 100 tokens reserved for distribution, an impressive 90% have already been snapped up!
yxson, @puncha, @breno-jesus-fernandes, @cleitonhoffmann, @vkoukoutsas, @CorentinDCAP, @AAnyass, @amitsgh, @adityamahendrap, @Hydrog3n, @pc-97, @deabrunner, @pseudoankit, @spenceryang, @Froler27, @wapele, @kishennagaraju, @wzhliang, @CalebCheptumo, @konradmadej
